src/colp/components/usetype_version_comparison_report.py

```diff
@@ -0,0 +1,76 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
+import plotly.express as px
+from st_aggrid import AgGrid
+from src.constants import COLOR_SCHEME
+from abc import ABC
+import pdb
+
+
+class UsetypeVersionComparisonReport:
+    def __init__(self, data):
+        self.usetype_changes = data["usetype_changes"]
+
+    def __call__(self):
+        st.subheader("Usetype Version Comparison")
+
+        version = st.sidebar.selectbox(
```

Comment:
Could this be made more readable if the code to select a version was moved to `colp.py` and this class was passed a version in it's `__init__`? That's more in line with my conception of how components exist on a page. Don't know if we've talked about this directly or how our other pages work but I wanted to raise the question

Reply:
Sure, I think that's a good assumption